README.md

```diff
@@ -11,8 +11,9 @@
 - Clone the repository
 - Replace YOUR_USERNAME and YOUR_ACCESS_KEY with your BrowserStack access credentials in browserstack.yml.
 - Install dependencies `mvn compile`
-- To run the test suite having cross-platform with parallelization, run `mvn test -P sample-test`
-- To run local tests, run `mvn test -P sample-local-test`
+- To run the test suite locally, run `mvn test -P sample-test`
```

Comment:
Let's not mess with this, create a new name for local profile. Example : `local-test` | `user-test` | `sample-test-local`

Reply:
This is decided internally